packages/shared/src/utils.ts

```diff
@@ -1,5 +1,6 @@
 import { base32hexnopad } from '@scure/base';
 import type { CodeLanguageType } from './types/cells.js';
+import * as crypto from 'crypto';
```

Comment:
This is used on the client in addition to the Node backend. What happens in that environment? Is vite smart enough to remove this from the compiled code, or does it try to do some sort of polyfill (e.g. the way webpack used to with crypto-browserify)?

Reply:
This should technically work Vite doesn't do a polyfil but ignores the import which *should* then cause it to end up using the global window crypto instance, or polyfills can be added if for some reason it's not but it does seem to be working

Reply:
I do not want us to use polyfills for something that exists natively. So assuming this drops the import then I'm stoked to get this working in node 18.